rosmon_core/src/monitor/node_monitor.cpp

```diff
@@ -559,7 +560,22 @@ void NodeMonitor::gatherCoredump(int signal)
 
 	// If the pattern is not absolute, it is relative to our node's cwd.
 	if(coreGlob[0] != '/')
-		coreGlob = m_processWorkingDirectory + "/" + coreGlob;
+		coreGlob = m_processWorkingDirectory.find("/tmp/rosmon-node-") == std::string::npos ? m_processWorkingDirectory + "/core_dumps/" + coreGlob : m_processWorkingDirectory + "/" + coreGlob;
```

Comment:
please verify it works by checking both flows (production and non production), and the core file getting generated can be parsed.
Also check if running `mon launch` and using the gdb mode also works (normal mon launch flow)

Reply:
Verified that it works